tests/queryset_pickle/tests.py

```diff
@@ -115,6 +115,38 @@ def test_pickle_prefetch_related_idempotence(self):
         groups = pickle.loads(pickle.dumps(groups))
         self.assertSequenceEqual(groups, [g])
 
+    def test_pickle_prefetch_queryset_not_evaluated(self):
```

Comment:
Simon/Adam, do you think we should include non-regression tests in a separate commit (in this case and/or in general) or do you think it's pointless?